sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/StreamExecution.scala

```diff
@@ -222,21 +228,6 @@ abstract class StreamExecution(
       }
     }
 
-  /**
```

Comment:
@jackierwzhang Sorry, shall we retain the code comment as it is when you move this and below?

Reply:
I removed it because they are duplicated with that in the class?